scripts/compare_pr_diff_binables.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# cleanup if needed
```

Comment:
Would be good to change the name of the file to include 'CI': we don't want the team to accidentally run this instead of the python script and get an unexpected `git clean`.

Reply:
We can look for a `CIRCLE` environment variable so nothing gets run outside of CI.

Can also change the filename, of course.